pkg/apis/kops/validation/aws.go

```diff
@@ -67,6 +67,8 @@ func awsValidateInstanceGroup(ig *kops.InstanceGroup, cloud awsup.AWSCloud) fiel
 
 	allErrs = append(allErrs, awsValidateInstanceType(field.NewPath(ig.GetName(), "spec", "machineType"), ig.Spec.MachineType, cloud)...)
```

Comment:
Does this make sense anymore? No reason to check the instance type in 2 places.

Reply:
Agreed we can push it to the arch instance type validation function. Will update this

Reply:
We should also check against all instance types in a mixed instance policy in addition to spec.machineType